### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by kito-cheng**
Describe this repo and navigation for related repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# RISC-V ELF psABI Document
+Processor specific application binary interface document for RISC-V.
```
Comment (jrtc27) [inline README.md:2]:
```suggestion
Processor-specific application binary interface document for RISC-V.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This document including follwoing items:
```
Comment (jrtc27) [inline README.md:4]:
```suggestion
This document includes the following items:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- DWARF Debug Information Format
+- Code Model
+- Relocation
+- Relaxation
+
+# Navigation
+----------------------------------------------------------------------------------------------------------
+Name                                | URL                                                   | Description
+------------------------------------|-------------------------------------------------------|--------------
+RISC-V ELF psABI                    | https://github.com/riscv/riscv-elf-psabi-doc          | Processor specific application binary interface document.
```
Comment (jrtc27) [inline README.md:16]:
```suggestion
RISC-V ELF psABI                    | https://github.com/riscv/riscv-elf-psabi-doc          | Processor-specific application binary interface document.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+RISC-V C API                        | https://github.com/riscv/riscv-c-api-doc              | RISC-V specific predefined macros, function attribute and  language extension.
```
Comment (jrtc27) [inline README.md:17]:
```suggestion
RISC-V C API                        | https://github.com/riscv/riscv-c-api-doc              | RISC-V-specific predefined macros, function attributes and language extensions.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+RISC-V Assembly Programmer's Manual | https://github.com/riscv/riscv-asm-manual             | Document for pseudoinstruction and assembly directive.
```
Comment (jrtc27) [inline README.md:18]:
```suggestion
RISC-V Assembly Programmer's Manual | https://github.com/riscv/riscv-asm-manual             | Document for pseudoinstructions and assembly directives.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+RISC-V Toolchain Conventions        | https://github.com/riscv/riscv-toolchain-conventions  | RISC-V specific toolchain behavior and command line option.
```
Comment (jrtc27) [inline README.md:19]:
```suggestion
RISC-V Toolchain Conventions        | https://github.com/riscv/riscv-toolchain-conventions  | RISC-V-specific toolchain behavior and command line option.
```

**Review by jrtc27 — COMMENTED** [review]
Thanks, just some minor grammatical tweaks

**Comment by kito-cheng** [comment]
ChangeLog:
- Apply suggestions.

**Comment by lazyparser** [comment]
good job. Finally the repo got its README :)

**Review by jim-wilson — APPROVED** [review]
The SBI isn't a compiler thing unlike the others, and has its own group, but I suppose people might get ABI and SBI confused so pointing them to the right place is OK, as long as it is clear that this group doesn't own it.

There is also EABI, but no useful or agreed upon spec yet, but you might want to point to it anyways.  https://github.com/riscv/riscv-eabi-spec

**Comment by kito-cheng** [comment]
Oh...EABI absolutely must list on here, I was struggling about SBI should appeared in the list or not, but I think it should be harmless to having more link in the list :P

ChangeLog:
- Add EABI.

**Comment by jrtc27** [comment]
Diffs in riscv-elf.md are weird, is that because you've ended up with a rebased copy of the commit on this branch too?

**Comment by kito-cheng** [comment]
@jrtc27 oh, my mistake, updated.

**Pull request merged**